dev-packages/application-manager/src/generator/webpack-generator.ts

```diff
@@ -45,6 +45,7 @@ module.exports = {
         path: outputPath
     },
     target: '${this.ifBrowser('web', 'electron-renderer')}',
+    mode: development ? 'development' : 'production',
```

Comment:
`development` should be computed based on `mode` option instead of `NODE_ENV` and fall back to `production` if it is not present, similar to webpack 4 defaults

Reply:
I don't think you can do that.

Reply:
why?

Reply:
The issue is that one can send `--mode production` now and set `NODE_ENV` to `development` which installs monaco sources. Although it should not be an issue if we migrate to the latest Monaco.

Reply:
webpack config is loaded before the arguments get reflected. We may be able to do that if we directly invoke webpack inside JS instead of spawning the CLI.

Reply:
It should be in `process.argv`.